CONTRIBUTING.md

```diff
@@ -26,6 +26,7 @@ Working on your first Pull Request? You can learn how from this *free* series [H
 - [Make Changes](#make-changes)
 - [Run The Test Suite](#run-the-test-suite)
 - [Squash Your Commits](#squash-your-commits)
+- [Commit messages](#commit-messages)
```

Comment:
For the sake of consistency, I guess this should be `Commit Message` instead of `Commit messages`